Database Query using PropertyFilter on Text returns error

Hi, I'm trying to query a database based on the property Name:
<img width="673" alt="image" src="https://user-images.githubusercontent.com/20089568/166626530-ec160b3c-e0f1-4a8d-852d-866621773fde.png">

The equivalent curl request gives a successful response:
```sh
➜ curl -X POST 'https://api.notion.com/v1/databases/'$DB_ID'/query' \
  -H 'Authorization: Bearer '"$NOTION_API_KEY"'' \
  -H 'Notion-Version: 2022-02-22' \
  -H "Content-Type: application/json" \
  --data '{
  "filter": {
    "property": "Name",
    "title": {
      "equals": "May 2022"
    }
  }
}'
```
However when using the package:
```go
	dbQueryReq := &notionapi.DatabaseQueryRequest{
		PropertyFilter: &notionapi.PropertyFilter{
			Property: "Name",
			Text: &notionapi.TextFilterCondition{
				Equals: today.Format("Jan 2006"),
			},
		},
	}
	d, err := t.client.Database.Query(ctx, notionapi.DatabaseID(dbId), dbQueryReq)
```
Returns the error
```
panic: body failed validation. Fix one:
body.filter.or should be defined, instead was `undefined`.
body.filter.title should be defined, instead was `undefined`.

...
```
Looks like the `Text` field in `PropertyFilter` needs to have it's json_tag updated to use `rich_text` ?

--- a/database_test.go
+++ b/database_test.go
@@ -173,7 +173,7 @@ func TestDatabaseClient(t *testing.T) {
 				request: &notionapi.DatabaseQueryRequest{
 					PropertyFilter: &notionapi.PropertyFilter{
 						Property: "Name",
-						Text: &notionapi.TextFilterCondition{
+						RichText: &notionapi.TextFilterCondition{
 							Contains: "Hel",
 						},
 					},

--- a/filter.go
+++ b/filter.go
@@ -7,8 +7,10 @@ type CompoundFilter map[FilterOperator][]PropertyFilter
 type Condition string
 
 type PropertyFilter struct {
-	Property    string                      `json:"property"`
+	Property string `json:"property"`
+	// DEPRECATED: doesn't for ver. 2022-02-22
 	Text        *TextFilterCondition        `json:"text,omitempty"`
+	RichText    *TextFilterCondition        `json:"rich_text,omitempty"`
 	Number      *NumberFilterCondition      `json:"number,omitempty"`
 	Checkbox    *CheckboxFilterCondition    `json:"checkbox,omitempty"`
 	Select      *SelectFilterCondition      `json:"select,omitempty"`
@@ -38,8 +40,8 @@ type NumberFilterCondition struct {
 	LessThan             *float64 `json:"less_than,omitempty"`
 	GreaterThanOrEqualTo *float64 `json:"greater_than_or_equal_to,omitempty"`
 	LessThanOrEqualTo    *float64 `json:"less_than_or_equal_to,omitempty"`
-	IsEmpty              bool    `json:"is_empty,omitempty"`
-	IsNotEmpty           bool    `json:"is_not_empty,omitempty"`
+	IsEmpty              bool     `json:"is_empty,omitempty"`
+	IsNotEmpty           bool     `json:"is_not_empty,omitempty"`
 }
 
 type CheckboxFilterCondition struct {

--- a/http/examples.http
+++ b/http/examples.http
@@ -4,14 +4,23 @@
 GET https://api.notion.com/v1/databases
 Content-Type: application/json
 Authorization: Bearer {{token}}
-Notion-Version: 2021-08-16
+Notion-Version: {{api_version}}
 
 ###
 # Query
-POST https://api.notion.com/v1/databases/b382c100-1fea-48b3-a503-72e86583ee4c/query
+POST https://api.notion.com/v1/databases/{{database_id}}/query
 Content-Type: application/json
 Authorization: Bearer {{token}}
 Notion-Version: {{api_version}}
+
+{
+  "filter": {
+    "property": "Name",
+    "rich_text": {
+      "equals": "Hello"
+    }
+  }
+}
 
 ###
 # Create

